src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let res = 1;
+
+  res = days * 40;
+
+  if (days >= 3 && days < 7) {
+    res -= 20;
```

Comment:
let's create variables for 20, 40, and 50 with meaningful names (should explain what it is)
because now it could be unclear